public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="/open-iconic/font/css/open-iconic-bootstrap.css" rel="stylesheet">
+    <link href="%PUBLIC_URL/main.css" rel="stylesheet">
```

Comment:
@akre96 you are missing a `%` here, which generates a `URIError: Failed to decode param '/%PUBLIC_URL/main.css'` error:

```suggestion
    <link href="%PUBLIC_URL%/main.css" rel="stylesheet">
```